Sources/Kitura/Kitura.swift

```diff
@@ -22,14 +22,31 @@ import Dispatch
 
 // MARK Kitura
 
-/// A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+/**
+ A set of helper functions to make it easier to create, start, and stop Kitura based servers.
+ ### Usage Example: ###
+ In this example, we create a function called run, inside a server `Application.swift` file, which will create a Kitura server on the specified port, using the given router and then run the created server. This function would then be the last call in your `main.swift` file.
```

Comment:
We need some more wording around the sentence 'This function would then be the last call in your main.swift file.' as it doesn't make sense just stuck at the end of this paragraph.